lambdas/question/src/main/java/uk/gov/di/ipv/cri/kbv/api/service/ServiceRegistry.java

```diff
@@ -0,0 +1,90 @@
+package uk.gov.di.ipv.cri.kbv.api.service;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import software.amazon.awssdk.services.sqs.SqsClient;
+import uk.gov.di.ipv.cri.common.library.persistence.DataStore;
+import uk.gov.di.ipv.cri.common.library.persistence.DynamoDbEnhancedClientFactory;
+import uk.gov.di.ipv.cri.common.library.persistence.item.SessionItem;
+import uk.gov.di.ipv.cri.common.library.service.AuditEventFactory;
+import uk.gov.di.ipv.cri.common.library.service.AuditService;
+import uk.gov.di.ipv.cri.common.library.service.ConfigurationService;
+import uk.gov.di.ipv.cri.common.library.service.PersonIdentityService;
+import uk.gov.di.ipv.cri.common.library.service.SessionService;
+import uk.gov.di.ipv.cri.common.library.util.ListUtil;
+import uk.gov.di.ipv.cri.kbv.api.gateway.KBVGatewayFactory;
+import uk.gov.di.ipv.cri.kbv.api.gateway.KeyStoreLoader;
+
+import java.time.Clock;
+
+public class ServiceRegistry {
+    private final ConfigurationService configurationService;
+    private AuditService auditService;
+    private SessionService sessionService;
+    private Clock clock = Clock.systemUTC();
+    private boolean isLoaded;
+
+    public ServiceRegistry(ConfigurationService configurationService) {
+        this.configurationService = configurationService;
+    }
+
+    public KBVService getKBVService() {
+        return new KBVService(new KBVGatewayFactory().create(this.configurationService));
+    }
+
+    public KBVStorageService getKBVStorageService() {
+        return new KBVStorageService(this.configurationService);
+    }
+
+    boolean isLoaded() {
+        return isLoaded;
+    }
+
+    void setLoaded(boolean loaded) {
+        isLoaded = loaded;
+    }
+
+    public void loadKeyStore() {
+        if (!isLoaded()) this.configureKeyStore();
+
+        setLoaded(true);
+    }
+
+    private void configureKeyStore() {
+        new KeyStoreLoader(this.configurationService).load();
+    }
+
+    public PersonIdentityService getPersonIdentityService() {
+        return new PersonIdentityService();
+    }
+
+    public AuditService getAuditService() {
+        if (auditService == null) {
+            auditService =
+                    new AuditService(
+                            SqsClient.builder().build(),
+                            this.configurationService,
+                            new ObjectMapper(),
+                            new AuditEventFactory(this.configurationService, clock));
+        }
+        return auditService;
+    }
+
+    public SessionService getSessionService() {
+        if (sessionService == null) {
+            sessionService =
+                    new SessionService(
+                            getSessionTableName(),
+                            this.configurationService,
+                            clock,
+                            new ListUtil());
+        }
+        return sessionService;
+    }
+
+    private DataStore<SessionItem> getSessionTableName() {
```

Comment:
this method can be removed 
it won't be needed with the new SessionService constructor you created